platform/src/main/java/org/hillview/sketches/MinKSet.java

```diff
@@ -0,0 +1,51 @@
+package org.hillview.sketches;
+
+import it.unimi.dsi.fastutil.longs.Long2ObjectRBTreeMap;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+/**
+ *  A data structure that computes the minHash of a column of a table. It stores the k column values
+ *  that hash to the minimum value.
+ */
+public class MinKSet {
+    public final int maxSize;
+    public final Long2ObjectRBTreeMap data;
```

Comment:
Is there a version of this class which takes Strings?
The code as written here is not type-safe.